draft-ietf-quic-qpack.md

```diff
@@ -291,7 +300,7 @@ An encoder MUST limit the number of streams which could become blocked to the
 value of SETTINGS_QPACK_BLOCKED_STREAMS at all times. Note that the decoder
 might not actually become blocked on every stream which risks becoming blocked.
 If the decoder encounters more blocked streams than it promised to support, it
-SHOULD treat this as a stream error of type HTTP_QPACK_DECOMPRESSION_FAILED.
+MUST treat this as an error of type HTTP_QPACK_DECOMPRESSION_FAILED.
```

Comment:
Same argument as above: I am worried that decoder implementations that do not do what they SHOULD do will allow the proliferation of non-conformant encoder implementations.

Reply:
I'm inclined to agree with you. Will let the MAY/SHOULD side weigh in.